packages/base/any/kernels/5.10-lts/patches/0024-i2c-Add-mv64xxx-support-for-AXI-bus-clock-handling.patch

```diff
@@ -0,0 +1,109 @@
+From 5604c64109bcd315c5d2e5461b7304f361d93642 Mon Sep 17 00:00:00 2001
```

Comment:
Where is that commit upstream?

    $ git show 5604c64109bcd315c5d2e5461b7304f361d93642
    fatal: bad object 5604c64109bcd315c5d2e5461b7304f361d93642